pkg/controller/protobuf/client/client.go

```diff
@@ -232,7 +235,7 @@ func (ctrlAdapter *controllerAdapter) run(ctx context.Context) {
 
 		interval := ctrlAdapter.backoff.NextBackOff()
 
-		logger.Printf("restarting controller in %s", interval)
+		logger.Sugar().Debugf("restarting controller in %s", interval)
```

Comment:
this might be `Infof` at least as this is pretty important to debug why controller doesn't reconcile

Reply:
yeah. I just moved to debug everything that was producing too much noise, but if it's critical then let's move it to Info.

Reply:
updated